blockchain/tx_pool.go

```diff
@@ -1550,22 +1578,23 @@ func (pool *TxPool) demoteUnexecutables() {
 			}
 		}
 
-		// TODO : Need to KIP-71 hardfork.
 		// Enqueue transaction if gasPrice of transaction is lower than gasPrice of txPool.
 		// All transactions with a nonce greater than enqueued transaction also stored queue.
-		if list.Len() > 0 {
-			needRemoved := false
-			for _, tx := range list.Flatten() {
-				hash := tx.Hash()
-				if needRemoved || tx.GasPrice().Cmp(pool.gasPrice) < 0 {
-					if needRemoved {
-						logger.Trace("Demoting the transaction due to the previous transaction that has lower gas price than baseFee", "txhash", hash)
-					} else {
-						needRemoved = true
-						logger.Trace("Demoting the transaction that has lower gas price than baseFee", "txhash", hash)
+		if pool.kip71 {
```

Comment:
Can you explain why `if pool.kip71` condition check should be here?
Is it not necessary to execute the logic below before the hard fork?

Reply:
Yes, It is added in https://github.com/klaytn/klaytn/pull/1382/files#diff-4a20f160f5182ec35d76a3566352906f577b84fbf3b242b922c4d2fc9c219cd7R1553.